Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://forum.adguard.com/index.php?threads/45115/223037
```

Comment:
This link does not exist on my end. What happened?)

Reply:
I think this is what you wanted to add https://forum.adguard.com/index.php?threads/resolved-amplitude-com-blocked.45115/